contracts/src/v0.8/dev/automation/2_1/interfaces/AutomationRegistryInterface2_1.sol

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+// TODO - I'm not sure we really need this anymore - might be nice to consolodate all the structs into Base
```

Comment:
it will be great if we put all the structs in base going forward. these structs have been floating in interfaces, logic contracts, base, etc.